src/JuliusSweetland.OptiKey/Services/AutoComplete/BasicAutoComplete.cs

```diff
@@ -50,24 +50,21 @@ public IEnumerable<string> GetSuggestions(string root)
         }
 
         public void AddEntry(string entry, DictionaryEntry newEntryWithUsageCount)
-        {
-
-            //Also add to entries for auto complete
-            var autoCompleteHash = entry.Normalise(log: false);
+		{
+			if (!string.IsNullOrWhiteSpace(entry) && entry.Contains(" "))
```

Comment:
Was there a reason for switching around the logic to add the new entry as a phrase and as a normal entry?